addons/sys_core/fnc_enableRevealAI.sqf

```diff
@@ -61,14 +61,14 @@ DFUNC(monitorAI_PFH) = {
                 if (_lastRevealed + 15 < time) then {
                     TRACE_2("Calling reveal event", _curUnit, _hasRevealed);
                     _curUnit setVariable [QGVAR(lastRevealed), time, false];
-                    [QGVAR(onRevealUnit), [acre_player, _curUnit, ACRE_REVEAL_AMOUNT ] ] call CALLSTACK(CBA_fnc_globalEvent);
+                    [QGVAR(onRevealUnit), [acre_player, _curUnit, ACRE_REVEAL_AMOUNT], _curUnit] call CALLSTACK(CBA_fnc_targetEvent);
                 };
             };
         };
     } forEach _nearUnits;
 
-    if (GVAR(revealToAI) isEqualTo 0) then {
-        [(_this select 1)] call CBA_fnc_removePerFrameHandler;
```

Comment:
What is the reason for going to `==` instead of `isEqualTo`?

Reply:
There was some discussion somewhere  (maybe ace slack?) that comparing numbers was still faster with ==. Jonpas reminded me of it in another PR

Reply:
```
commy2 [19:25]
 Oh, and `!=` is faster than `!(isEqualTo)`.

commy2 [19:30]
 The only reason you should actively use IET is case sensitive comparission, the different interpretation of null, and comparing arrays.
Best with a comment explaining why for the first two of these.
Oh, or if the two data types could be different.
```